ui/components/DataTable.tsx

```diff
@@ -98,7 +98,7 @@ export const sortWithType = (rows: Row[], sort: Field) => {
         else return 1;
 
       default:
-        return sortFn(a).localeCompare(sortFn(b));
+        return (sortFn(a) || "").localeCompare(sortFn(b));
```

Comment:
I think the `|| ''` also has to go on `sortFn(b)`

Reply:
It probably wouldn't have exploded, but added it anyway